### PR TITLE
Interface

### PR DESCRIPTION
**Pull request by fherrerao**
# Hi There 💯 
### Features implemented in this branch

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person's id.

Please let me know if I should make changes

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        puts "Date: #{rental.date} Book: #{rental.book.title} by #{rental.book.author}"
+      end
+    end
+  end
+end
+
+def main
+  app = App.new
+  app.run
+end
```
Comment (mateo951) [inline app.rb:153]: Great! Your app entry point is on your main method 🔥

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+main
```
Comment (mateo951) [inline app.rb:155]: Fantastic✔ main method is invoked at the end of the main.rb file

**Review by mateo951 — APPROVED** [review]
### Hi @fherrerao, 

👀 **Your project highlights** 👀
- General Requirements ✔
- Fantastic PR 🔥
- Your interface works perfectly 💪😎 Congrats!

Your project is complete!  There is nothing else to say other than... it's time to merge it 🙌 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (fherrerao) [reply app.rb:155]: Thanks for the time that you spent reviewing my project, I appreciate it.

**Pull request merged**